easybuild/easyconfigs/g/GTK+/GTK+-2.24.31-intel-2017a.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'ConfigureMake'
+
+name = 'GTK+'
+version = '2.24.31'
+
+homepage = 'https://developer.gnome.org/gtk+/stable/'
+description = """
+ The GTK+ 2 package contains libraries used for creating graphical user interfaces for applications.
+"""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+source_urls = [FTPGNOME_SOURCE]
+sources = [SOURCELOWER_TAR_XZ]
+
+dependencies = [
+    ('ATK', '2.22.0'),
+    ('Gdk-Pixbuf', '2.36.0'),
+    ('Pango', '1.40.3'),
```

Comment:
an easyconfig for Pango 1.40.5 is already merged, shall we bump this version and avoid future headaches with conflicts on dependency versions?